src/articles/list.tsx

```diff
@@ -107,5 +145,8 @@ export function ArticleList(props: ArticleListProps, renderer: RendererLike<Node
         }/>
       </div>
     </div>
+    <Conditional if={props.isLoading ?? state(false)}
```

Comment:
you can use `of(false)` instead of `state(false)` for a single value that never changes.